src/standard/gasometer/mod.rs

```diff
@@ -366,6 +366,7 @@ fn dynamic_opcode_cost<H: RuntimeBackend>(
 				target_is_cold: handler.is_cold(address, Some(index)),
 			}
 		}
+		Opcode::TLOAD if config.eip_1153_enabled => GasCost::TLoad,
```

Comment:
The gas cost of `TLOAD` and `TSTORE` opcode should all be `100`

Reply:
It was the case, but I do agree that using `gas_storage_read_warm` is the correct way to do it. Done.